install.ps1

```diff
@@ -661,12 +825,66 @@ refreshenv
 Write-Host "[+] Installing shared module..."
 choco install common.vm -y --force
 refreshenv
+Start-Sleep 1
 
-# Save the modified config to where the installer will look for it
-Write-Host "[+] Saving modified configuration file..."
-$configXml.save((Join-Path ${Env:VM_COMMON_DIR} "config.xml"))
+# Log basic system info to assist future troubleshooting
```

Comment:
Should this go into a separate file/cmd (see https://github.com/mandiant/VM-Packages/issues/49)?

Reply:
See: https://github.com/mandiant/VM-Packages/pull/110